src/commands/deploy/deploy.ts

```diff
@@ -690,40 +682,52 @@ const printResults = ({
     logJson(jsonData)
     exit(0)
   } else {
+    const message = deployToProduction
+      ? `Deployed to production URL: ${terminalLink(results.siteUrl, results.siteUrl)}\n
+    Unique deploy URL: ${terminalLink(results.deployUrl, results.deployUrl)}`
+      : `Deployed draft to ${terminalLink(results.deployUrl, results.deployUrl)}`
+
+    log(
+      boxen(message, {
+        padding: 1,
+        margin: 1,
+        textAlignment: 'center',
+        borderStyle: 'round',
+        borderColor: NETLIFY_CYAN_HEX,
+        // This is an intentional half-width space to work around a unicode padding math bug in boxen
+        // eslint-disable-next-line no-irregular-whitespace
+        title: `⬥  ${deployToProduction ? 'Production deploy' : 'Draft deploy'} is live ⬥ `,
+        titleAlignment: 'center',
+      }),
+    )
+
     log(prettyjson.render(msgData))
 
     if (!deployToProduction) {
       log()
-      log('If everything looks good on your draft URL, deploy it to your main site URL with the --prod flag.')
-      log(
-        chalk.cyanBright.bold(
-          `netlify ${isIntegrationDeploy ? 'integration:' : ''}deploy${runBuildCommand ? ' --build' : ''} --prod`,
-        ),
-      )
+      log('If everything looks good on your draft URL, deploy it to your main site URL with the --prod flag:')
+      log(chalk.cyanBright.bold(`netlify deploy${runBuildCommand ? '' : '--no-build'} --prod`))
```

Comment:
this is unrelated, just a little thing we missed in https://github.com/netlify/cli/pull/7195